zhaquirks/linxura/__init__.py

```diff
@@ -0,0 +1,57 @@
+"""Linxura button device."""
+
+from zigpy.quirks import CustomCluster
+import zigpy.zcl.clusters.security
+
+from zhaquirks.const import (
+    BUTTON,
+    BUTTON_1,
+    BUTTON_2,
+    BUTTON_3,
+    BUTTON_4,
+    COMMAND_DOUBLE,
+    COMMAND_HOLD,
+    COMMAND_ID,
+    COMMAND_PRESS,
+    PRESS_TYPE,
+    ZHA_SEND_EVENT,
+)
+
+LINXURA = "Linxura"
+STATUS_REPORT = 2
+
+
+PRESS_TYPES = {
+    1: COMMAND_PRESS,
+    2: COMMAND_DOUBLE,
+    3: COMMAND_HOLD,
+}
+
+
+class LinxuraIASCluster(CustomCluster, zigpy.zcl.clusters.security.IasZone):
+    """Occupancy cluster."""
+
+    def _update_attribute(self, attrid, value):
+        self.info("Linxura update attribute - attrid: %d, value:%d", attrid, value)
+        super()._update_attribute(attrid, value)
+        if attrid == STATUS_REPORT:
```

Comment:
Remove the constant `STATUS_REPORT` and replace it with getting the id of `zone_status` like this:
```suggestion
        if attrid == self.AttributeDefs.zone_status.id:
```
(or `IasZone.AttributeDefs.zone_status.id`)

Reply:
Thank you! Modified based on your kind suggestion.